tests/test_mvt.py

```diff
@@ -81,9 +81,10 @@ def _check_metatile(self, metatile_size):
             for (posargs, kwargs), coord in zip(encode.call_args_list,
                                                 tile_coords):
                 self.assertIn('quantize_bounds', kwargs)
-                quantize_bounds = kwargs['quantize_bounds']
-                extent = int(round((quantize_bounds[2] - quantize_bounds[0]) /
-                                   resolution))
+                # quantize_bounds = kwargs['quantize_bounds']
```

Comment:
Delete these instead of commenting?

Reply:
> Delete these instead of commenting?

we hardcoded it to be 4096, but a better logic would be to use some of the commented out calculation; so maybe we should keep them? I added more comments/doc for this in the next commit.